servicebox-app/src/main/java/io/halkyon/services/ClaimingJobService.java

```diff
@@ -0,0 +1,70 @@
+package io.halkyon.services;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.transaction.Transactional;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import io.halkyon.model.Claim;
+import io.halkyon.model.ClaimingStatus;
+import io.halkyon.model.Service;
+import io.quarkus.scheduler.Scheduled;
+
+@ApplicationScoped
+public class ClaimingJobService {
```

Comment:
Could we have a javadoc describing the job algorithm? Same for the methods of the class....

Reply:
Added

Reply:
PR updated